builds/misc/packages-release.yaml

```diff
@@ -987,14 +994,33 @@ stages:
                 secretsFilter: >-
                   GitHubAccessToken
             - checkout: self
+            - checkout: azure-iotedge
+              submodules: recursive
             - bash: |
                 echo "Approval Complete"
 
+                BASE_VERSION="$(cat $BUILD_SOURCESDIRECTORY/iotedge/edgelet/version.txt)"
```

Comment:
I think you need to remove 'iotedge' from the path specified here.

Reply:
Nope. If there are more 1 repository checked out, it will use the default name of the repository as a directory. 
This `deploy` has 2 repositories checked out `/iotedge (self)` and `azure-iotedge`. Both will be put under `$(Build.SourcesDirectory)` e.g. `$(Build.SourcesDirectory)/iotedge` and `$(Build.SourcesDirectory)/azure-iotedge`